doc/Makefile

```diff
@@ -90,5 +90,5 @@ endif
 
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(OPTS) is meant as a shortcut for $(SPHINXOPTS).
-%: Makefile doxy
+%: Makefile
```

Comment:
I do not understand this change, can you explain?

Also, since one needs to run `make html` first, wouldn't it be better to define a new phony target and make it depend on the html target? I.e.:
```
.PHONY: latexpdf
latexpdf: html
```
```

Reply:
How about also adding `xindy` to ``doc/tutorials/docbuild.rst` directly?

Reply:
As for removing the doxy dependency here, doxy wasn't sufficient for building with the unmentioned builders, so I was being a bit lazy and just say build html first and then build latexpdf.  (I'm still figuring out how to get rid of the PDF-building errors and was running it multiple times, so no need to repeat the html step every time I wanted to rerun the latexpdf step.  Once we get the latexpdf more cleanly working we can consolidate this into one step that creates the input needed to do PDF generation that's done now for the html generation.

Reply:
OK, makes sense. Thanks!